build.gradle

```diff
@@ -38,6 +38,41 @@ startScripts {
     }
 }
 
+final String buildPrerequisitesErrorFormatString = "%s See https://github.com/broadinstitute/gatk#building for information on how to build GATK."
+
+// Large runtime resource files must be present at build time to be compiled into the jar, so validate
+// that the lfs files have been fetched and that the directory does not contain the git-lfs pointer files
+def validateLargeResourceFile(final File largeResourceFile, buildPrerequisitesErrorFormatString) {
+    largeResourceFile.withReader("UTF-8") { reader ->
+        final String firstLine = reader.readLine();
+        if (firstLine != null && firstLine.startsWith("version https://git-lfs.github.com/spec/v1")) {
```

Comment:
Declare this string as a named constant somewhere

Reply:
done.